aion/web/elm/elm-package.json

```diff
@@ -20,7 +20,8 @@
         "krisajenkins/remotedata": "4.3.0 <= v < 5.0.0",
         "lgastako/elm-select": "2.0.0 <= v < 3.0.0",
         "rundis/elm-bootstrap": "3.0.0 <= v < 4.0.0",
-        "pablen/toasty": "1.0.4 <= v <= 1.0.4"
```

Comment:
The fact that there are no trailing commas allowed in elixir really grinds my gears.

Reply:
Its not elixir, thats just JSON format mate :)

Reply:
Yeah right, sorry, that's just late night me. Anyway this is also what I experienced while swapping deps in mix.exs a lot lately, so seeing the deps list, I didn't hesitate before blaming elixir 😆